AENS.md

```diff
@@ -297,18 +303,25 @@ A `claim` transaction MUST NOT be in included in the same block as its
  ------------ ----
 | client_ttl | 8  |
  ------------ ----
+| rent       | 32 |
+ ------------ ----
 | pointers   |    |
  ------------ ----
 ```
 
-The `update` transaction MUST be signed by the owner
-of the name entry to be updated.
+The `update` transaction MUST be signed by the owner of the name entry to be
+updated.
 
-The `expire_by` MUST NOT be more than 36000 blocks into
-the future.
+The `expire_by` MUST NOT be more than `floor(60 * 24 * (365/2)/key_block_interval)`,
```

Comment:
The [code limits this relative TTL to 50000 blocks](https://github.com/aeternity/epoch/blob/e9e3b000475b58501fc0daf8e8b1d57ea8b25669/apps/aens/src/aens_update_tx.erl#L241-L242). Is this specified anywhere?

Reply:
That specific limit is not specified anywhere.